authors/apps/authentication/backends.py

```diff
@@ -1,9 +1,84 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from .models import User
+
 
 """Configure JWT Here"""
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    auth_header_prefix = 'Bearer'.lower()
+
+    def authenticate(self, request):
+        """
+        This method will be called everytime an endpoint is accessed.
+
+        This method can return 'None' when we want authentication to
+        fail, for example when no authenticaiton credentials are provided.
```

Comment:
@MandelaK, please correct the spelling of `authentication`.

Reply:
Done